spec/post_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  first_user = User.create(Name: 'Tom', Photo: 'https://unsplash.com/photos/F_-0BxGuVvo', Bio: 'Teacher from Mexico.')
+  subject { Post.new(AuthorId: first_user.id, Title: 'Hello', Text: 'This is my first post') }
+  before { subject.save }
+
+  it 'title length should not be too long' do
+    subject.Title = 'a' * 300
+    expect(subject).to_not be_valid
+  end
+
+  it 'author_id should be present' do
+    subject.AuthorId = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'comments counter should be integer' do
+    subject.CommentsCounter = 'String'
+    expect(subject).to_not be_valid
```

Comment:
- [ ] Great job working on the tests but I think you missed testing the method `update_post_counter`  and `recent_comments` in the Post but since it is found in models so you should test it as shown in the requirements so kindly add test to the method ✨✅🔥

![image](https://user-images.githubusercontent.com/58553711/215096423-fbb8a9dd-0d87-4c7b-9e17-b994b5341eaf.png)